screens/leaderboard/GlobalLeaderboardScreen.tsx

```diff
@@ -0,0 +1,91 @@
+import { Ionicons } from '@expo/vector-icons';
+import { useCallback, useEffect, useState } from 'react';
+import { SafeAreaView, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { useLazyGetGlobalLeaderboardQuery } from '../../components/user/userApi';
+import LeaderboardList from './LeaderboardList';
+
+const DEFAULT_PAGE = 0;
+const DEFAULT_PAGE_SIZE = 10;
+
+const GlobalLeaderboardScreen = () => {
+  const [currentPage, setCurrentPage] = useState(DEFAULT_PAGE);
+  const [totalPageNum, setTotalPageNum] = useState(0);
+  const [isRefreshing, setIsRefreshing] = useState(false);
+  const [trigger, { data: leaderboard, isFetching, isError }] = useLazyGetGlobalLeaderboardQuery();
+
+  const goToPreviousPage = () => {
+    if (currentPage > 0) {
```

Comment:
Comments for the `FriendLeaderboardScreen` apply here as well. Please apply the same changes.